build.gradle

```diff
@@ -21,21 +21,21 @@ dependencies {
     compile 'io.socket:engine.io-client:0.7.0'
     testCompile group: 'junit', name: 'junit', version: '4.11'
     testCompile "org.mockito:mockito-core:1.+"
-    cucumberCompile  "info.cukes:cucumber-java:1.2.4"
-    cucumberCompile  "info.cukes:cucumber-junit:1.2.4"
+    testCompile  "info.cukes:cucumber-java:1.2.4"
+    testCompile  "info.cukes:cucumber-junit:1.2.4"
```

Comment:
Not sure why I couldn't get this working. Didn't investigate for too long but can look into it if need be